lesson4/_homework/task_01/script.js

```diff
@@ -0,0 +1,22 @@
+
+//                    Adapt Academy 2019 lesson 04 - JS | task 01 by Donatas Dereškevičius                     
+
+const countryList = [ 
+    {"name": "Afghanistan", "code": "AF"}, 
+    {"name": "Åland Islands", "code": "AX"}, 
+    {"name": "Albania", "code": "AL"}, 
+    {"name": "Algeria", "code": "DZ"}
+    ]
+
+    const getCountryByCode = (data) => {
+
+        input = document.querySelector(".form-input").value;
+        data = input.toUpperCase();
```

Comment:
You should not reassign a function parameter. Better way would be doing something like this: `const dataUpperCase = input.toUpperCase();`